src/worker/core/phase/newPhaseBeforeDraft.ts

```diff
@@ -50,6 +50,46 @@ const newPhaseBeforeDraft = async (
 			await idb.cache.players.put(p);
 		}
 	}
+	const challengeLoseBestPlayer = g.get("challengeLoseBestPlayer");
+	if (challengeLoseBestPlayer) {
+		const tids = g.get("userTids");
+		for (let i = 0; i < tids.length; i++) {
+			const tid = tids[i];
+			const [playersAll] = await Promise.all([
+				idb.cache.players.indexGetAll("playersByTid", tid),
+			]);
```

Comment:
Can just be  `const playersAll = await idb.cache.players.indexGetAll("playersByTid", tid);`